src/app/modules/customer-management/components/projects/components/create-project/create-project.component.ts

```diff
@@ -107,4 +109,9 @@ export class CreateProjectComponent implements OnInit, OnDestroy {
       project_type_id: null,
     });
   }
+
+  onSearchChanges(searchValue: string): void {
```

Comment:
This function is identical to `onSearchChanges` of create-customer.ts
[Here](https://github.com/ioet/time-tracker-ui/blob/6d8b0b5003c99bb4f607f25c902af726e52b3c3c/src/app/modules/customer-management/components/customer-info/components/create-customer/create-customer.ts#L101)